spec/unit/shoulda/matchers/active_model/allow_value_matcher_spec.rb

```diff
@@ -9,6 +9,15 @@
 end
 
 describe Shoulda::Matchers::ActiveModel::AllowValueMatcher, type: :model do
+  describe "backwards compatibility" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.